docs/configuration-reference/components/web-ui.md

```diff
@@ -46,25 +47,25 @@ Table of all the arguments accepted by the component.
 Example:
```

Comment:
This line is not needed.

Reply:
True, I'll remove it and file another PR to remove similar lines from other docs.